### ISSUE TITLE
Choice of encoding causes program to crash

### ISSUE THREAD

**Issue by jglazer63** [RandomEngy/VidCoder#952]
### Problem Description

Choosing QSV, VCN or MF hardware accelerated encoding causes the program to simply crash.  No error. No nothing.  Just gone.  NVec works but the others cause a crash.

### What version of VidCoder are you running?

6.44 Installer

### Encode Log

```text
# VidCoder 6.44
# Starting scan: N:\Foundation - S01E08 - The Missing Piece.mkv
9:20] Intel Quick Sync Video support: no
[11:59:20] hb_scan: path=N:\Foundation - S01E08 - The Missing Piece.mkv, title_index=0
udfread ERROR: ECMA 167 Volume Recognition failed
src/libbluray/disc/disc.c:333: failed opening UDF image N:\Foundation - S01E08 - The Missing Piece.mkv
src/libbluray/disc/disc.c:437: error opening file BDMV\index.bdmv
src/libbluray/disc/disc.c:437: error opening file BDMV\BACKUP\index.bdmv
src/libbluray/bluray.c:2646: nav_get_title_list(N:\Foundation - S01E08 - The Missing Piece.mkv\) failed
[11:59:20] bd: not a bd - trying as a stream/file instead
libdvdread: DVDOpenFileUDF:UDFFindFile /VIDEO_TS/VIDEO_TS.IFO failed
libdvdnav: vm: vm: failed to read VIDEO_TS.IFO
[11:59:20] dvd: not a dvd - trying as a stream/file instead
Input #0, matroska,webm, from 'N:\Foundation - S01E08 - The Missing Piece.mkv':
  Metadata:
    encoder         : libebml v1.4.0 + libmatroska v1.6.1
  Duration: 00:56:17.18, start: 0.000000, bitrate: 3616 kb/s
  Stream #0:0(eng): Video: h264 (High), yuv420p(tv, bt709, progressive), 1356x678 [SAR 1:1 DAR 2:1], 23.98 fps, 23.98 tbr, 1k tbn, 47.95 tbc (default)
  Stream #0:1(eng): Audio: eac3, 48000 Hz, 5.1(side), fltp, 768 kb/s (default)
  Stream #0:2(eng): Subtitle: subrip (default) (forced)
    Metadata:
      title           : English [Forced]
  Stream #0:3(eng): Subtitle: subrip
    Metadata:
      title           : English
  Stream #0:4(ara): Subtitle: subrip
    Metadata:
      title           : Arabic
  Stream #0:5(bul): Subtitle: subrip
    Metadata:
      title           : Bulgarian
  Stream #0:6(chi): Subtitle: subrip
    Metadata:
      title           : Chinese Simplified
  Stream #0:7(chi): Subtitle: subrip
    Metadata:
      title           : Chinese Traditional
  Stream #0:8(chi): Subtitle: subrip
    Metadata:
      title           : Chinese Yue
  Stream #0:9(cze): Subtitle: subrip
    Metadata:
      title           : Czech
  Stream #0:10(dan): Subtitle: subrip
    Metadata:
      title           : Danish
  Stream #0:11(dut): Subtitle: subrip
    Metadata:
      title           : Dutch
  Stream #0:12(est): Subtitle: subrip
    Metadata:
      title           : Estonian
  Stream #0:13(spa): Subtitle: subrip
    Metadata:
      title           : European Spanish
  Stream #0:14(fin): Subtitle: subrip
    Metadata:
      title           : Finnish
  Stream #0:15(fre): Subtitle: subrip
    Metadata:
      title           : French
  Stream #0:16(fre): Subtitle: subrip
    Metadata:
      title           : French Canadian
  Stream #0:17(ger): Subtitle: subrip
    Metadata:
      title           : German
  Stream #0:18(gre): Subtitle: subrip
    Metadata:
      title           : Greek
  Stream #0:19(heb): Subtitle: subrip
    Metadata:
      title           : Hebrew
  Stream #0:20(hin): Subtitle: subrip
    Metadata:
      title           : Hindi
  Stream #0:21(hun): Subtitle: subrip
    Metadata:
      title           : Hungarian
  Stream #0:22(ind): Subtitle: subrip
    Metadata:
      title           : Indonesian
  Stream #0:23(ita): Subtitle: subrip
    Metadata:
      title           : Italian
  Stream #0:24(jpn): Subtitle: subrip
    Metadata:
      title           : Japanese
  Stream #0:25(kor): Subtitle: subrip
    Metadata:
      title           : Korean
  Stream #0:26(spa): Subtitle: subrip
    Metadata:
      title           : Latin America Spanish
  Stream #0:27(lav): Subtitle: subrip
    Metadata:
      title           : Latvian
  Stream #0:28(lit): Subtitle: subrip
    Metadata:
      title           : Lithuanian
  Stream #0:29(may): Subtitle: subrip
    Metadata:
      title           : Malay
  Stream #0:30(nor): Subtitle: subrip
    Metadata:
      title           : Norwegian
  Stream #0:31(pol): Subtitle: subrip
    Metadata:
      title           : Polish
  Stream #0:32(por): Subtitle: subrip
    Metadata:
      title           : Portuguese
  Stream #0:33(por): Subtitle: subrip
    Metadata:
      title           : Portuguese Brazilian
  Stream #0:34(rus): Subtitle: subrip
    Metadata:
      title           : Russian
  Stream #0:35(slo): Subtitle: subrip
    Metadata:
      title           : Slovak
  Stream #0:36(slv): Subtitle: subrip
    Metadata:
      title           : Slovenian
  Stream #0:37(swe): Subtitle: subrip
    Metadata:
      title           : Swedish
  Stream #0:38(tam): Subtitle: subrip
    Metadata:
      title           : Tamil
  Stream #0:39(tel): Subtitle: subrip
    Metadata:
      title           : Telugu
  Stream #0:40(tha): Subtitle: subrip
    Metadata:
      title           : Thai
  Stream #0:41(tur): Subtitle: subrip
    Metadata:
      title           : Turkish
  Stream #0:42(ukr): Subtitle: subrip
    Metadata:
      title           : Ukrainian
  Stream #0:43(vie): Subtitle: subrip
    Metadata:
      title           : Vietnamese
[11:59:20] scan: decoding previews for title 1
[11:59:20] scan: audio 0x1: eac3, rate=48000Hz, bitrate=768000 English (E-AC3) (5.1 ch) (768 kbps)
[11:59:21] scan: 10 previews, 1356x678, 23.976 fps, autocrop = 0/0/0/0, aspect 2.00:1, PAR 1:1, color profile: 1-1-1
[11:59:21] libhb: scan thread found 1 valid title(s)
# Titles JSON:
{
    "MainFeature": 0,
    "TitleList": [
        {
            "AngleCount": 1,
            "AudioList": [
                {
                    "Attributes": {
                        "AltCommentary": false,
                        "Commentary": false,
                        "Default": true,
                        "Normal": false,
                        "Secondary": false,
                        "VisuallyImpaired": false
                    },
                    "BitRate": 768000,
                    "ChannelCount": 6,
                    "ChannelLayout": 1551,
                    "ChannelLayoutName": "5.1(side)",
                    "Codec": 16777216,
                    "CodecName": "eac3",
                    "CodecParam": 86056,
                    "Description": "English (E-AC3) (5.1 ch) (768 kbps)",
                    "LFECount": 1,
                    "Language": "English",
                    "LanguageCode": "eng",
                    "SampleRate": 48000,
                    "TrackNumber": 1
                }
            ],
            "ChapterList": [
                {
                    "Duration": {
                        "Hours": 0,
                        "Minutes": 56,
                        "Seconds": 17,
                        "Ticks": 303946560
                    },
                    "Name": ""
                }
            ],
            "Color": {
                "Format": 0,
                "Matrix": 1,
                "Primary": 1,
                "Range": 1,
                "Transfer": 1
            },
            "Container": "matroska,webm",
            "Crop": [
                0,
                0,
                0,
                0
            ],
            "Duration": {
                "Hours": 0,
                "Minutes": 56,
                "Seconds": 17,
                "Ticks": 303946560
            },
            "FrameRate": {
                "Den": 1126125,
                "Num": 27000000
            },
            "Geometry": {
                "Height": 678,
                "PAR": {
                    "Den": 1,
                    "Num": 1
                },
                "Width": 1356
            },
            "Index": 1,
            "InterlaceDetected": false,
            "Metadata": {},
            "Name": "Foundation - S01E08 - The Missing Piece",
            "Path": "N:\\Foundation - S01E08 - The Missing Piece.mkv",
            "Playlist": -1,
            "SubtitleList": [
                {
                    "Attributes": {
                        "4By3": false,
                        "Children": false,
                        "ClosedCaption": false,
                        "Commentary": false,
                        "Default": true,
                        "Forced": true,
                        "Large": false,
                        "Letterbox": false,
                        "Normal": false,
                        "PanScan": false,
                        "Wide": false
                    },
                    "Format": "text",
                    "Language": "English [UTF-8]",
                    "LanguageCode": "eng",
                    "Name": "English [Forced]",
                    "Source": 3,
                    "SourceName": "UTF-8",
                    "TrackNumber": 1
                },
                {
                    "Attributes": {
                        "4By3": false,
                        "Children": false,
                        "ClosedCaption": false,
                        "Commentary": false,
                        "Default": false,
                        "Forced": false,
                        "Large": false,
                        "Letterbox": false,
                        "Normal": false,
                        "PanScan": false,
                        "Wide": false
                    },
                    "Format": "text",
                    "Language": "English [UTF-8]",
                    "LanguageCode": "eng",
                    "Name": "English",
                    "Source": 3,
                    "SourceName": "UTF-8",
                    "TrackNumber": 2
                },
                {
                    "Attributes": {
                        "4By3": false,
                        "Children": false,
                        "ClosedCaption": false,
                        "Commentary": false,
                        "Default": false,
                        "Forced": false,
                        "Large": false,
                        "Letterbox": false,
                        "Normal": false,
                        "PanScan": false,
                        "Wide": false
                    },
                    "Format": "text",
                    "Language": "العربية [UTF-8]",
                    "LanguageCode": "ara",
                    "Name": "Arabic",
                    "Source": 3,
                    "SourceName": "UTF-8",
                    "TrackNumber": 3
                },
                {
                    "Attributes": {
                        "4By3": false,
                        "Children": false,
                        "ClosedCaption": false,
                        "Commentary": false,
                        "Default": false,
                        "Forced": false,
                        "Large": false,
                        "Letterbox": false,
                        "Normal": false,
                        "PanScan": false,
                        "Wide": false
                    },
                    "Format": "text",
                    "Language": "български [UTF-8]",
                    "LanguageCode": "bul",
                    "Name": "Bulgarian",
                    "Source": 3,
                    "SourceName": "UTF-8",
                    "TrackNumber": 4
                },
                {
                    "Attributes": {
                        "4By3": false,
                        "Children": false,
                        "ClosedCaption": false,
                        "Commentary": false,
                        "Default": false,
                        "Forced": false,
                        "Large": false,
                        "Letterbox": false,
                        "Normal": false,
                        "PanScan": false,
                        "Wide": false
                    },
                    "Format": "text",
                    "Language": "中文 [UTF-8]",
                    "LanguageCode": "zho",
                    "Name": "Chinese Simplified",
                    "Source": 3,
                    "SourceName": "UTF-8",
                    "TrackNumber": 5
                },
                {
                    "Attributes": {
                        "4By3": false,
                        "Children": false,
                        "ClosedCaption": false,
                        "Commentary": false,
                        "Default": false,
                        "Forced": false,
                        "Large": false,
                        "Letterbox": false,
                        "Normal": false,
                        "PanScan": false,
                        "Wide": false
                    },
                    "Format": "text",
                    "Language": "中文 [UTF-8]",
                    "LanguageCode": "zho",
                    "Name": "Chinese Traditional",
                    "Source": 3,
                    "SourceName": "UTF-8",
                    "TrackNumber": 6
                },
                {
                    "Attributes": {
                        "4By3": false,
                        "Children": false,
                        "ClosedCaption": false,
                        "Commentary": false,
                        "Default": false,
                        "Forced": false,
                        "Large": false,
                        "Letterbox": false,
                        "Normal": false,
                        "PanScan": false,
                        "Wide": false
                    },
                    "Format": "text",
                    "Language": "中文 [UTF-8]",
                    "LanguageCode": "zho",
                    "Name": "Chinese Yue",
                    "Source": 3,
                    "SourceName": "UTF-8",
                    "TrackNumber": 7
                },
                {
                    "Attributes": {
                        "4By3": false,
                        "Children": false,
                        "ClosedCaption": false,
                        "Commentary": false,
                        "Default": false,
                        "Forced": false,
                        "Large": false,
                        "Letterbox": false,
                        "Normal": false,
                        "PanScan": false,
                        "Wide": false
                    },
                    "Format": "text",
                    "Language": "čeština [UTF-8]",
                    "LanguageCode": "ces",
                    "Name": "Czech",
                    "Source": 3,
                    "SourceName": "UTF-8",
                    "TrackNumber": 8
                },
                {
                    "Attributes": {
                        "4By3": false,
                        "Children": false,
                        "ClosedCaption": false,
                        "Commentary": false,
                        "Default": false,
                        "Forced": false,
                        "Large": false,
                        "Letterbox": false,
                        "Normal": false,
                        "PanScan": false,
                        "Wide": false
                    },
                    "Format": "text",
                    "Language": "dansk [UTF-8]",
                    "LanguageCode": "dan",
                    "Name": "Danish",
                    "Source": 3,
                    "SourceName": "UTF-8",
                    "TrackNumber": 9
                },
                {
                    "Attributes": {
                        "4By3": false,
                        "Children": false,
                        "ClosedCaption": false,
                        "Commentary": false,
                        "Default": false,
                        "Forced": false,
                        "Large": false,
                        "Letterbox": false,
                        "Normal": false,
                        "PanScan": false,
                        "Wide": false
                    },
                    "Format": "text",
                    "Language": "Nederlands [UTF-8]",
                    "LanguageCode": "nld",
                    "Name": "Dutch",
                    "Source": 3,
                    "SourceName": "UTF-8",
                    "TrackNumber": 10
                },
                {
                    "Attributes": {
                        "4By3": false,
                        "Children": false,
                        "ClosedCaption": false,
                        "Commentary": false,
                        "Default": false,
                        "Forced": false,
                        "Large": false,
                        "Letterbox": false,
                        "Normal": false,
                        "PanScan": false,
                        "Wide": false
                    },
                    "Format": "text",
                    "Language": "eesti [UTF-8]",
                    "LanguageCode": "est",
                    "Name": "Estonian",
                    "Source": 3,
                    "SourceName": "UTF-8",
                    "TrackNumber": 11
                },
                {
                    "Attributes": {
                        "4By3": false,
                        "Children": false,
                        "ClosedCaption": false,
                        "Commentary": false,
                        "Default": false,
                        "Forced": false,
                        "Large": false,
                        "Letterbox": false,
                        "Normal": false,
                        "PanScan": false,
                        "Wide": false
                    },
                    "Format": "text",
                    "Language": "español [UTF-8]",
                    "LanguageCode": "spa",
                    "Name": "European Spanish",
                    "Source": 3,
                    "SourceName": "UTF-8",
                    "TrackNumber": 12
                },
                {
                    "Attributes": {
                        "4By3": false,
                        "Children": false,
                        "ClosedCaption": false,
                        "Commentary": false,
                        "Default": false,
                        "Forced": false,
                        "Large": false,
                        "Letterbox": false,
                        "Normal": false,
                        "PanScan": false,
                        "Wide": false
                    },
                    "Format": "text",
                    "Language": "suomi [UTF-8]",
                    "LanguageCode": "fin",
                    "Name": "Finnish",
                    "Source": 3,
                    "SourceName": "UTF-8",
                    "TrackNumber": 13
                },
                {
                    "Attributes": {
                        "4By3": false,
                        "Children": false,
                        "ClosedCaption": false,
                        "Commentary": false,
                        "Default": false,
                        "Forced": false,
                        "Large": false,
                        "Letterbox": false,
                        "Normal": false,
                        "PanScan": false,
                        "Wide": false
                    },
                    "Format": "text",
                    "Language": "Francais [UTF-8]",
                    "LanguageCode": "fra",
                    "Name": "French",
                    "Source": 3,
                    "SourceName": "UTF-8",
                    "TrackNumber": 14
                },
                {
                    "Attributes": {
                        "4By3": false,
                        "Children": false,
                        "ClosedCaption": false,
                        "Commentary": false,
                        "Default": false,
                        "Forced": false,
                        "Large": false,
                        "Letterbox": false,
                        "Normal": false,
                        "PanScan": false,
                        "Wide": false
                    },
                    "Format": "text",
                    "Language": "Francais [UTF-8]",
                    "LanguageCode": "fra",
                    "Name": "French Canadian",
                    "Source": 3,
                    "SourceName": "UTF-8",
                    "TrackNumber": 15
                },
                {
                    "Attributes": {
                        "4By3": false,
                        "Children": false,
                        "ClosedCaption": false,
                        "Commentary": false,
                        "Default": false,
                        "Forced": false,
                        "Large": false,
                        "Letterbox": false,
                        "Normal": false,
                        "PanScan": false,
                        "Wide": false
                    },
                    "Format": "text",
                    "Language": "Deutsch [UTF-8]",
                    "LanguageCode": "deu",
                    "Name": "German",
                    "Source": 3,
                    "SourceName": "UTF-8",
                    "TrackNumber": 16
                },
                {
                    "Attributes": {
                        "4By3": false,
                        "Children": false,
                        "ClosedCaption": false,
                        "Commentary": false,
                        "Default": false,
                        "Forced": false,
                        "Large": false,
                        "Letterbox": false,
                        "Normal": false,
                        "PanScan": false,
                        "Wide": false
                    },
                    "Format": "text",
                    "Language": "Ελληνικά [UTF-8]",
                    "LanguageCode": "ell",
                    "Name": "Greek",
                    "Source": 3,
                    "SourceName": "UTF-8",
                    "TrackNumber": 17
                },
                {
                    "Attributes": {
                        "4By3": false,
                        "Children": false,
                        "ClosedCaption": false,
                        "Commentary": false,
                        "Default": false,
                        "Forced": false,
                        "Large": false,
                        "Letterbox": false,
                        "Normal": false,
                        "PanScan": false,
                        "Wide": false
                    },
                    "Format": "text",
                    "Language": "עברית [UTF-8]",
                    "LanguageCode": "heb",
                    "Name": "Hebrew",
                    "Source": 3,
                    "SourceName": "UTF-8",
                    "TrackNumber": 18
                },
                {
                    "Attributes": {
                        "4By3": false,
                        "Children": false,
                        "ClosedCaption": false,
                        "Commentary": false,
                        "Default": false,
                        "Forced": false,
                        "Large": false,
                        "Letterbox": false,
                        "Normal": false,
                        "PanScan": false,
                        "Wide": false
                    },
                    "Format": "text",
                    "Language": "हिंदी [UTF-8]",
                    "LanguageCode": "hin",
                    "Name": "Hindi",
                    "Source": 3,
                    "SourceName": "UTF-8",
                    "TrackNumber": 19
                },
                {
                    "Attributes": {
                        "4By3": false,
                        "Children": false,
                        "ClosedCaption": false,
                        "Commentary": false,
                        "Default": false,
                        "Forced": false,
                        "Large": false,
                        "Letterbox": false,
                        "Normal": false,
                        "PanScan": false,
                        "Wide": false
                    },
                    "Format": "text",
                    "Language": "magyar [UTF-8]",
                    "LanguageCode": "hun",
                    "Name": "Hungarian",
                    "Source": 3,
                    "SourceName": "UTF-8",
                    "TrackNumber": 20
                },
                {
                    "Attributes": {
                        "4By3": false,
                        "Children": false,
                        "ClosedCaption": false,
                        "Commentary": false,
                        "Default": false,
                        "Forced": false,
                        "Large": false,
                        "Letterbox": false,
                        "Normal": false,
                        "PanScan": false,
                        "Wide": false
                    },
                    "Format": "text",
                    "Language": "Bahasa Indonesia [UTF-8]",
                    "LanguageCode": "ind",
                    "Name": "Indonesian",
                    "Source": 3,
                    "SourceName": "UTF-8",
                    "TrackNumber": 21
                },
                {
                    "Attributes": {
                        "4By3": false,
                        "Children": false,
                        "ClosedCaption": false,
                        "Commentary": false,
                        "Default": false,
                        "Forced": false,
                        "Large": false,
                        "Letterbox": false,
                        "Normal": false,
                        "PanScan": false,
                        "Wide": false
                    },
                    "Format": "text",
                    "Language": "italiano [UTF-8]",
                    "LanguageCode": "ita",
                    "Name": "Italian",
                    "Source": 3,
                    "SourceName": "UTF-8",
                    "TrackNumber": 22
                },
                {
                    "Attributes": {
                        "4By3": false,
                        "Children": false,
                        "ClosedCaption": false,
                        "Commentary": false,
                        "Default": false,
                        "Forced": false,
                        "Large": false,
                        "Letterbox": false,
                        "Normal": false,
                        "PanScan": false,
                        "Wide": false
                    },
                    "Format": "text",
                    "Language": "日本語 [UTF-8]",
                    "LanguageCode": "jpn",
                    "Name": "Japanese",
                    "Source": 3,
                    "SourceName": "UTF-8",
                    "TrackNumber": 23
                },
                {
                    "Attributes": {
                        "4By3": false,
                        "Children": false,
                        "ClosedCaption": false,
                        "Commentary": false,
                        "Default": false,
                        "Forced": false,
                        "Large": false,
                        "Letterbox": false,
                        "Normal": false,
                        "PanScan": false,
                        "Wide": false
                    },
                    "Format": "text",
                    "Language": "한국어 [UTF-8]",
                    "LanguageCode": "kor",
                    "Name": "Korean",
                    "Source": 3,
                    "SourceName": "UTF-8",
                    "TrackNumber": 24
                },
                {
                    "Attributes": {
                        "4By3": false,
                        "Children": false,
                        "ClosedCaption": false,
                        "Commentary": false,
                        "Default": false,
                        "Forced": false,
                        "Large": false,
                        "Letterbox": false,
                        "Normal": false,
                        "PanScan": false,
                        "Wide": false
                    },
                    "Format": "text",
                    "Language": "español [UTF-8]",
                    "LanguageCode": "spa",
                    "Name": "Latin America Spanish",
                    "Source": 3,
                    "SourceName": "UTF-8",
                    "TrackNumber": 25
                },
                {
                    "Attributes": {
                        "4By3": false,
                        "Children": false,
                        "ClosedCaption": false,
                        "Commentary": false,
                        "Default": false,
                        "Forced": false,
                        "Large": false,
                        "Letterbox": false,
                        "Normal": false,
                        "PanScan": false,
                        "Wide": false
                    },
                    "Format": "text",
                    "Language": "latviešu [UTF-8]",
                    "LanguageCode": "lav",
                    "Name": "Latvian",
                    "Source": 3,
                    "SourceName": "UTF-8",
                    "TrackNumber": 26
                },
                {
                    "Attributes": {
                        "4By3": false,
                        "Children": false,
                        "ClosedCaption": false,
                        "Commentary": false,
                        "Default": false,
                        "Forced": false,
                        "Large": false,
                        "Letterbox": false,
                        "Normal": false,
                        "PanScan": false,
                        "Wide": false
                    },
                    "Format": "text",
                    "Language": "lietuvių [UTF-8]",
                    "LanguageCode": "lit",
                    "Name": "Lithuanian",
                    "Source": 3,
                    "SourceName": "UTF-8",
                    "TrackNumber": 27
                },
                {
                    "Attributes": {
                        "4By3": false,
                        "Children": false,
                        "ClosedCaption": false,
                        "Commentary": false,
                        "Default": false,
                        "Forced": false,
                        "Large": false,
                        "Letterbox": false,
                        "Normal": false,
                        "PanScan": false,
                        "Wide": false
                    },
                    "Format": "text",
                    "Language": "Unknown [UTF-8]",
                    "LanguageCode": "und",
                    "Name": "Malay",
                    "Source": 3,
                    "SourceName": "UTF-8",
                    "TrackNumber": 28
                },
                {
                    "Attributes": {
                        "4By3": false,
                        "Children": false,
                        "ClosedCaption": false,
                        "Commentary": false,
                        "Default": false,
                        "Forced": false,
                        "Large": false,
                        "Letterbox": false,
                        "Normal": false,
                        "PanScan": false,
                        "Wide": false
                    },
                    "Format": "text",
                    "Language": "norsk [UTF-8]",
                    "LanguageCode": "nor",
                    "Name": "Norwegian",
                    "Source": 3,
                    "SourceName": "UTF-8",
                    "TrackNumber": 29
                },
                {
                    "Attributes": {
                        "4By3": false,
                        "Children": false,
                        "ClosedCaption": false,
                        "Commentary": false,
                        "Default": false,
                        "Forced": false,
                        "Large": false,
                        "Letterbox": false,
                        "Normal": false,
                        "PanScan": false,
                        "Wide": false
                    },
                    "Format": "text",
                    "Language": "polski [UTF-8]",
                    "LanguageCode": "pol",
                    "Name": "Polish",
                    "Source": 3,
                    "SourceName": "UTF-8",
                    "TrackNumber": 30
                },
                {
                    "Attributes": {
                        "4By3": false,
                        "Children": false,
                        "ClosedCaption": false,
                        "Commentary": false,
                        "Default": false,
                        "Forced": false,
                        "Large": false,
                        "Letterbox": false,
                        "Normal": false,
                        "PanScan": false,
                        "Wide": false
                    },
                    "Format": "text",
                    "Language": "Portugues [UTF-8]",
                    "LanguageCode": "por",
                    "Name": "Portuguese",
                    "Source": 3,
                    "SourceName": "UTF-8",
                    "TrackNumber": 31
                },
                {
                    "Attributes": {
                        "4By3": false,
                        "Children": false,
                        "ClosedCaption": false,
                        "Commentary": false,
                        "Default": false,
                        "Forced": false,
                        "Large": false,
                        "Letterbox": false,
                        "Normal": false,
                        "PanScan": false,
                        "Wide": false
                    },
                    "Format": "text",
                    "Language": "Portugues [UTF-8]",
                    "LanguageCode": "por",
                    "Name": "Portuguese Brazilian",
                    "Source": 3,
                    "SourceName": "UTF-8",
                    "TrackNumber": 32
                },
                {
                    "Attributes": {
                        "4By3": false,
                        "Children": false,
                        "ClosedCaption": false,
                        "Commentary": false,
                        "Default": false,
                        "Forced": false,
                        "Large": false,
                        "Letterbox": false,
                        "Normal": false,
                        "PanScan": false,
                        "Wide": false
                    },
                    "Format": "text",
                    "Language": "русский [UTF-8]",
                    "LanguageCode": "rus",
                    "Name": "Russian",
                    "Source": 3,
                    "SourceName": "UTF-8",
                    "TrackNumber": 33
                },
                {
                    "Attributes": {
                        "4By3": false,
                        "Children": false,
                        "ClosedCaption": false,
                        "Commentary": false,
                        "Default": false,
                        "Forced": false,
                        "Large": false,
                        "Letterbox": false,
                        "Normal": false,
                        "PanScan": false,
                        "Wide": false
                    },
                    "Format": "text",
                    "Language": "slovenčina [UTF-8]",
                    "LanguageCode": "slk",
                    "Name": "Slovak",
                    "Source": 3,
                    "SourceName": "UTF-8",
                    "TrackNumber": 34
                },
                {
                    "Attributes": {
                        "4By3": false,
                        "Children": false,
                        "ClosedCaption": false,
                        "Commentary": false,
                        "Default": false,
                        "Forced": false,
                        "Large": false,
                        "Letterbox": false,
                        "Normal": false,
                        "PanScan": false,
                        "Wide": false
                    },
                    "Format": "text",
                    "Language": "slovenščina [UTF-8]",
                    "LanguageCode": "slv",
                    "Name": "Slovenian",
                    "Source": 3,
                    "SourceName": "UTF-8",
                    "TrackNumber": 35
                },
                {
                    "Attributes": {
                        "4By3": false,
                        "Children": false,
                        "ClosedCaption": false,
                        "Commentary": false,
                        "Default": false,
                        "Forced": false,
                        "Large": false,
                        "Letterbox": false,
                        "Normal": false,
                        "PanScan": false,
                        "Wide": false
                    },
                    "Format": "text",
                    "Language": "svenska [UTF-8]",
                    "LanguageCode": "swe",
                    "Name": "Swedish",
                    "Source": 3,
                    "SourceName": "UTF-8",
                    "TrackNumber": 36
                },
                {
                    "Attributes": {
                        "4By3": false,
                        "Children": false,
                        "ClosedCaption": false,
                        "Commentary": false,
                        "Default": false,
                        "Forced": false,
                        "Large": false,
                        "Letterbox": false,
                        "Normal": false,
                        "PanScan": false,
                        "Wide": false
                    },
                    "Format": "text",
                    "Language": "தமிழ் [UTF-8]",
                    "LanguageCode": "tam",
                    "Name": "Tamil",
                    "Source": 3,
                    "SourceName": "UTF-8",
                    "TrackNumber": 37
                },
                {
                    "Attributes": {
                        "4By3": false,
                        "Children": false,
                        "ClosedCaption": false,
                        "Commentary": false,
                        "Default": false,
                        "Forced": false,
                        "Large": false,
                        "Letterbox": false,
                        "Normal": false,
                        "PanScan": false,
                        "Wide": false
                    },
                    "Format": "text",
                    "Language": "తెలుగు [UTF-8]",
                    "LanguageCode": "tel",
                    "Name": "Telugu",
                    "Source": 3,
                    "SourceName": "UTF-8",
                    "TrackNumber": 38
                },
                {
                    "Attributes": {
                        "4By3": false,
                        "Children": false,
                        "ClosedCaption": false,
                        "Commentary": false,
                        "Default": false,
                        "Forced": false,
                        "Large": false,
                        "Letterbox": false,
                        "Normal": false,
                        "PanScan": false,
                        "Wide": false
                    },
                    "Format": "text",
                    "Language": "ไทย [UTF-8]",
                    "LanguageCode": "tha",
                    "Name": "Thai",
                    "Source": 3,
                    "SourceName": "UTF-8",
                    "TrackNumber": 39
                },
                {
                    "Attributes": {
                        "4By3": false,
                        "Children": false,
                        "ClosedCaption": false,
                        "Commentary": false,
                        "Default": false,
                        "Forced": false,
                        "Large": false,
                        "Letterbox": false,
                        "Normal": false,
                        "PanScan": false,
                        "Wide": false
                    },
                    "Format": "text",
                    "Language": "Türkçe [UTF-8]",
                    "LanguageCode": "tur",
                    "Name": "Turkish",
                    "Source": 3,
                    "SourceName": "UTF-8",
                    "TrackNumber": 40
                },
                {
                    "Attributes": {
                        "4By3": false,
                        "Children": false,
                        "ClosedCaption": false,Q
                        "Commentary": false,
                        "Default": false,
                        "Forced": false,
                        "Large": false,
                        "Letterbox": false,
                        "Normal": false,
                        "PanScan": false,
                        "Wide": false
                    },
                    "Format": "text",
                    "Language": "українська [UTF-8]",
                    "LanguageCode": "ukr",
                    "Name": "Ukrainian",
                    "Source": 3,
                    "SourceName": "UTF-8",
                    "TrackNumber": 41
                },
                {
                    "Attributes": {
                        "4By3": false,
                        "Children": false,
                        "ClosedCaption": false,
                        "Commentary": false,
                        "Default": false,
                        "Forced": false,
                        "Large": false,
                        "Letterbox": false,
                        "Normal": false,
                        "PanScan": false,
                        "Wide": false
                    },
                    "Format": "text",
                    "Language": "Tiếng Việt [UTF-8]",
                    "LanguageCode": "vie",
                    "Name": "Vietnamese",
                    "Source": 3,
                    "SourceName": "UTF-8",
                    "TrackNumber": 42
                }
            ],
            "Type": 3,
            "VideoCodec": "h264"
        }
    ]
}
x264 [error]: invalid preset 'balanced'
# Scan completed
```

**Comment by RandomEngy** [comment]
Try it with HandBrake. VidCoder uses HandBrake for the core encoding engine, so if the issue reproduces there it's an issue with their code and you can ask them for help. If it doesn't reproduce with HandBrake, we know it's a VidCoder issue.

Equivalent versions:
VidCoder Stable ↔ [HandBrake Stable](https://handbrake.fr/downloads.php)
VidCoder Beta ↔ [HandBrake Nightly](https://handbrake.fr/nightly.php)

**Comment by jglazer63** [comment]
> Try it with HandBrake. VidCoder uses HandBrake for the core encoding engine, so if the issue reproduces there it's an issue with their code and you can ask them for help. If it doesn't reproduce with HandBrake, we know it's a VidCoder issue.
> 
> Equivalent versions: VidCoder Stable ↔ [HandBrake Stable](https://handbrake.fr/downloads.php) VidCoder Beta ↔ [HandBrake Nightly](https://handbrake.fr/nightly.php)

Thank you for the reply.  I see that handbrake has those encoders grayed out so I suppose my system does not support them.  On VidCoder they are not grayed out so I was able to choose them and crash the program.

**Comment by RandomEngy** [comment]
The HandBrake core DLL has an API to list all available encoders, which VidCoder is using. That core function will check if an encoder is listed as available. The official HandBrake Windows UI re-implements a lot of this logic, individually calling functions and allowing users to force-disable some encoders from settings. I didn't see any logic for graying out incompatible encoders. Their code was just removing them from the list entirely.

Can you post a screenshot of what you're seeing on HandBrake with grayed out encoders?

**Comment by jglazer63** [comment]
Ok, now when I run handbrake it does not show them grayed out, just entirely gone:
[https://paste.pics/ad68df29d07473c313780b275a462379](url)

However in VidCoder they are not being hidden or grayed out at all:
[https://paste.pics/30329e02fae0b1ae3930a15a93fffe40](url)

Hope this helps.  I do like the program a lot :)

**Comment by RandomEngy** [comment]
Ahh I see. The VidCoder screenshot is from the preset selection menu. HandBrake hides that away a bit more than VidCoder does. Theirs is under the Presets menu:

![image](https://user-images.githubusercontent.com/9450861/147584021-de3a2af5-c739-430a-8ee7-dbbf14d0ade0.png)

VidCoder automatically pulls in the HandBrake presets; I need to make sure I disable or remove them from the preset list. This will be fixed in the next release.

**Comment by jglazer63** [comment]
Sounds good.  Thanks!  Again, if, for whatever reason, you choose one of those, the program crashes.

**Comment by RandomEngy** [comment]
Should be fixed in v6.45 / v7.2 Beta. Presets with unsupported encoders won't be shown in the list.
